submissions/adeyemiJuwon/todo/src/components/Form.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { IoIosAdd } from 'react-icons/io';
+
+export default class AddTodo extends Component {
+  state = {
+    todo: '',
+  };
+  onChange = (e) => this.setState({ todo: e.target.value.toLowerCase() });
+  onSubmit = (e) => {
+    e.preventDefault();
+    this.props.addtodo(this.state.todo);
```

Comment:
Where is the function `addTodo`?

Reply:
Inside the App. js file and I pass it as a props to the AddTodo. js file. every of my function is inside the App.js while I pass it as props to their respective component

Reply:
@ToluAkin

Reply:
noted